toucan_connectors/github/github_connector.py

```diff
@@ -246,6 +252,21 @@ async def get_pages(
                 except ValueError:
                     pass
 
+            # For now we want to retrieve only max 1 year of Pull requests
+            # TODO change this to be able to receive a start date for extraction as a parameter
+            if dataset == GithubDataSet('pull requests'):
```

Comment:
ya de la logique spécifique aux pull requests qui se mélange ici
Je me dis que si ca continue et quil y a dautres cas, on pourrait reflechir à séparer un peu.
Par ex:

- if Pull Request
// logique pour couper lhistorique plus tot
  -> arret au dernier objet?
  -> arret à 1 an ?

Reply:
Merci pour la remarque. 
En effet, il y'a une logique spécifique pull requests. 
Comme il va y avoir une autre itération sur ce connecteur pour paramétrer la profondeur d'historique à récupérer. Je ferai cette refacto à ce moment là.